go.mod

```diff
@@ -0,0 +1,63 @@
+module kubevirt.io/node-maintenance-operator
+
+go 1.13
+
+require (
+	github.com/Azure/go-autorest/autorest v0.10.2 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.8.3 // indirect
+	github.com/appscode/jsonpatch v0.0.0-20190108182946-7c0e3b262f30 // indirect
+	github.com/coreos/prometheus-operator v0.0.0-00010101000000-000000000000 // indirect
+	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
+	github.com/go-logr/zapr v0.1.1 // indirect
+	github.com/go-openapi/spec v0.19.3
+	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
+	github.com/golang/mock v1.2.0
+	github.com/google/btree v1.0.0 // indirect
+	github.com/gophercloud/gophercloud v0.11.0 // indirect
+	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/imdario/mergo v0.3.9 // indirect
+	github.com/mitchellh/go-homedir v1.1.0 // indirect
+	github.com/onsi/ginkgo v1.11.0
+	github.com/onsi/gomega v1.7.0
+	github.com/operator-framework/operator-sdk v0.17.1-0.20200523015351-b602297bde06
+	github.com/pborman/uuid v1.2.0 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/prometheus/client_golang v1.6.0 // indirect
+	github.com/sergi/go-diff v1.1.0 // indirect
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/pflag v1.0.5
+	go.uber.org/zap v1.15.0 // indirect
+	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37 // indirect
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
+	golang.org/x/time v0.0.0-20200416051211-89c76fbcd5d1 // indirect
+	golang.org/x/tools v0.0.0-20200603131921-a45abac6c9c7 // indirect
+	k8s.io/api v0.18.3
+	k8s.io/apimachinery v0.18.3
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/klog v1.0.0
+	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
+	sigs.k8s.io/controller-runtime v0.1.12
+	sigs.k8s.io/controller-tools v0.1.10 // indirect
+	sigs.k8s.io/testing_frameworks v0.1.2 // indirect
+)
+
+// Pinned to kubernetes-1.13.4
```

Comment:
Why 1.13.4?  Gopkg was using 1.13.1 (see Gopkg.lock for all the versions in use)
At this stage we just want to change the way dependancies are managed.
The contents of the vendor directory, especially for k8s packages, should be basically identical.

Reply:
> Why 1.13.4? Gopkg was using 1.13.1 (see Gopkg.lock for all the versions in use)
> At this stage we just want to change the way dependancies are managed.
> The contents of the vendor directory, especially for k8s packages, should be basically identical.

this project is working with operator-sdk v0.8.0. I took that sdk version and generated an empty project with it - here we get a go.mod file that works with v0.8.0 (it includes the sigs controller, client-go, prometheus and k8s api and apimachinery packages). These are exactly the same dependencies we need for nmo. 

I tried to derive the go.mod file from the current definitions in Gopkg.lock - but that didn't succeed.

Reply:
Marc Sluiter got the trick of getting  smaller set changs, we got down to 443 changed files or so.